### ISSUE TITLE
Invalid parameter: 'ssh_keypath'

### ISSUE THREAD

**Issue by ltutar** [puppet-community/puppet-rundeck#150]
I have the following puppet code after downloading the latest version from github.
```
  rundeck::config::project { 'PetClinicProject':
    ssh_keypath             => '/opt/rundeck/key/insecure_private_key',
    resource_sources => {
      'PetClinicProject' => {
        project_name        => 'PetClinicProject',
        source_type         => 'directory',
        directory           => '/opt/rundeck/nodes',
        include_server_node => false,
        resource_format     => 'resourcejson',
      }
    }
  }
```

When I run it, I get the error:
```
[root@rundeckagent ~]# puppet agent -t
Info: Retrieving pluginfacts
Info: Retrieving plugin
Info: Loading facts
Error: Could not retrieve catalog from remote server: Error 400 on SERVER: Invalid parameter: 'ssh_keypath' on Rundeck::Config::Project[PetClinicProject] at /etc/puppetlabs/code/environments/production/modules/rundeck_config/manifests/config.pp:32 on node rundeckagent.home
Warning: Not using cache on failed catalog
Error: Could not retrieve catalog; skipping run
[root@rundeckagent ~]# 
```

This is the same as the example given in config/project.pp
```
# rundeck::config::project { 'test project':
#  ssh_keypath            => '/var/lib/rundeck/.ssh/id_rsa',
#  file_copier_provider   => 'jsch-scp',
#  node_executor_provider => 'jsch-ssh',
#  resource_sources       => $resource_hash
# }
```
Am I missing something here?

**Comment by jyaworski** [comment]
Hello:

This is a duplicate of #118 and will be solved once #153 is merged.
